src/buffer_manager.ts

```diff
@@ -581,7 +581,7 @@ export class BufferManager implements Disposable {
 
                 if (token?.isCancellationRequested) continue;
                 this.syncLayoutProgress.report("Cleaning up windows and buffers");
-                await this.cleanupWindowsAndBuffers(visibleEditors);
+                this.cleanupWindowsAndBuffers(visibleEditors);
```

Comment:
Here might be an issue; it is recommended to add a comment.

Reply:
Sure, what should the comment say? I guess I'm not sure what issue you are referring to here

Reply:
Ah, I thought that the comment I added inside `cleanupWindowsAndBuffers` would explain that, and I think I removed `await` here because the linter didn't like that I was awaiting a non-Promise. I can add something though